packages/replay/src/integration.ts

```diff
@@ -30,7 +30,7 @@ let _initialized = false;
 type InitialReplayPluginOptions = Omit<ReplayPluginOptions, 'sessionSampleRate' | 'errorSampleRate'> &
   Partial<Pick<ReplayPluginOptions, 'sessionSampleRate' | 'errorSampleRate'>>;
 
-export const replayIntegration = ((options?: InitialReplayPluginOptions) => {
+export const replayIntegration = ((options?: ReplayConfiguration) => {
```

Comment:
We'll want the options type to be

```
Omit<ReplayPluginOptions, 'sessionSampleRate' | 'errorSampleRate'>
```

because these sample rate options are deprecated and slated to be removed, can you confirm @mydea?

Reply:
Ok then, In that case i guess it should be wrapped with Partial then. 
Because the options are required in ReplayPluginOptions type. Causing the error mentioned.

Reply:
@Shubhdeep12 Ah my mistake, it should be `Omit<ReplayConfiguration, 'sessionSampleRate' | 'errorSampleRate'>`

Reply:
So I think `ReplayConfiguration` is correct for now, we can remove the deprecated fields in a follow up in v8!